documentation.md

```diff
@@ -29,9 +29,9 @@ Some documentation, such as local procedures, results of security audits, etc wi
 
 ## Restrictions
 
-The general restrictions set out in the [top-level open-source policy](https://github.com/dwp/dataworks-open-source-policy/blob/master/open_source_policy.md) apply universally for all open-source documentation. All of the restrictions set out in that document _must_ be adhered to for all documentation and designs.
+The general restrictions set out in the [top-level open-source policy](https://github.com/dwp/dataworks-open-source-policy/blob/master/open_source_policy.md) apply universally for all open-source information. All of the restrictions set out in that document _must_ be adhered to for all documentation and designs.
 
-A common challenges for documentation and designs is the tendency for network information such as IP addresses to creep into the designs.
+A common challenge for documentation and designs is the tendency for network information such as IP addresses to creep into the designs.
```

Comment:
That can be fixed with a sensible policy though, c.f. https://ministryofjustice.github.io/security-guidance/mythbusting/ip-dns-diagram-handling/#ip-addresses-dns-information--architecture-documentation

Reply:
:) although irrespective of whatever sensible views you might hold around IP addresses there are some genuinely sensitive types of information which can creep into design documentation un-noticed! Thanks for the review chap.